widget.md

```diff
@@ -0,0 +1,48 @@
+## Quick Start
+
+This embedded `<iframe>` allows you to embed a Harvest Timer form directly into your application. Create an `<iframe>` like this:
+
+```
+<iframe 
+  src="https://platform.harvestapp.com/platform/timer?
+    app_name=ExampleCompany&
+    closable=false&
+    permalink=https%3A%2F%2Fexample.com%2Fitem%2F1&
+    external_item_id=1&
```

Comment:
Yeah, I think I’m down with that. The only reason I can think to keep them is that they reinforce that the things are the third-party things and not the Harvest things, but (a) I don’t think it really works very well, since “external to who” isn’t very clear, and (b) `item` and `group` aren’t the same nouns as `project` and `task`, which makes this distinction clear enough already I think.

I’ll prep a PR to Harvestapp and then update these docs!